parlai/agents/transformer/modules.py

```diff
@@ -1031,7 +1131,17 @@ def prepare_head(tensor):
 
         out = self.out_lin(attentioned)
 
-        return out
+        return out, new_incr_state
+
+    @staticmethod
```

Comment:
Hm! Static method is a curious choice. Is there a reason why?

Reply:
Just because it's not using any class methods/attrs. I could make it a separate function outside the class, but I do think it belongs to MHA so `@staticmethod` makes sense to me here

Reply:
I’d prefer a normal method, even if it isn’t using the state, but it’s also fine this way

Reply:
Ah, I can see how a normal method would have increased parallelism with the `.reorder_incremental_state()` methods of other classes. Okay, changed